lib/taskana-cdi/pom.xml

```diff
@@ -25,29 +25,35 @@
 			<version>${project.version}</version>
 		</dependency>
 
+		<dependency>
+			<groupId>org.slf4j</groupId>
+			<artifactId>slf4j-simple</artifactId>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.slf4j</groupId>
+			<artifactId>jcl-over-slf4j</artifactId>
+			<scope>test</scope>
+		</dependency>
```

Comment:
what is this dependency for?

Reply:
fixed

Reply:
I made a mistake yesterday. This dependency is necessary. Can you please re-add it?

Reply:
reverted